ideas/122-sob-increase_platform_usage.md

```diff
@@ -0,0 +1,60 @@
+## Preamble
+
+    Idea: 122-sob-increase_platform_usage
+    Title: Open Bounty - Increase Platform Usage
```

Comment:
I think `OpenBounty — Platform Usage Metrics` and `122-sob-metrics` might be more appropriate names/identifiers for this idea/swarm as this swarm really would not work towards increasing usage but to improving our understanding about platform usage and general adoption.

Reply:
Good point.